initCobraToolbox.m

```diff
@@ -171,7 +171,7 @@ function initCobraToolbox()
 %If the Matlab version is prior to Matlab 2016a, we have to use the TC.xml
 %structure to provide autocompletion, otherwise we can use the json
 %definition.
-if MatlabYear < 2015
+if MatlabYear < 2016

```

Comment:
We could do an "else" statement after the setup for R2015 and below, that informs that autocomplete was added. BUt I don't think its necessary. its more an "expected behaviour" (my opinion)